Dungeoneer/src/com/interrupt/dungeoneer/game/Level.java

```diff
@@ -98,17 +98,30 @@
 	/** Ending distance of fog. */
 	public float fogEnd;
 
+    private static final Color defaultFogColor = new Color(0, 0, 0, 1);
```

Comment:
Mmh, same comment for above. Could this also be encapsulated in a default `LevelInfo` object? Or is that not needed? Also fine with the current approach.

Reply:
This is independent of LevelInfo object and represent the default value for the fog so it can be used in comparisons.